src/ex5/server/services/itemManager.js

```diff
@@ -0,0 +1,168 @@
+const pokemonClinet = require("../clients/pokemonClient.js");
+const { items } = require("../db/models");
+
+class ItemManager {
+  constructor() {
+    this.itemsArray = [];
+    this.taskToDbPath = "../server/db/taskToDb.json";
+  }
+
+  async getAll() {
+    try{
+      this.itemsArray = await items.findAll({raw : true});
+      return this.itemsArray;
+    }catch(err){
+      throw new Error(err)
+    }
+  }
+
+  async deleteAll() {
+    try{
+    this.itemsArray = [];
+    await items.destroy({
```

Comment:
a common solution is to have another field in the table `deleted` and marked it as true, as you don't want to lose data or let the user the ability to restore data.
it will require from you to add new field and fetch items that is `deleted: false.`